README.md

```diff
@@ -213,7 +241,7 @@ export const LoginPage = () => {
   }
 
   return (
-    <section>
+    <main className={`flex min-h-screen flex-col items-center justify-between p-24`}>
```

Comment:
We should skip `className`; it's unrelated to auth and only obscures what we're trying to teach here.

Reply:
@mmiszy it looks pretty bad without, but OK; could be confusing long term. I have another solution for that later